src/style.css

```
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.big-search {
+  position: relative;
+  height: 70px;
+}
+
+.icon {
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.input {
+  width: 100%;
+  height: 100%;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
```

```suggestion
  font-size: 16px;
  line-height: 22px;
  color: #3d4e61;
```
lets move to `html` and use inherit if there is a need

---

you can use `font-family: inherit;`